sig-docs/charter.md

```diff
@@ -0,0 +1,54 @@
+# SIG Docs Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and adheres to the Roles and Organization Management specified in [sig-governance].
+
+## Scope
+
+SIG Docs is responsible for Kubernetes documentation.
+
+The responsibility for feature documentation lies with feature developers. SIG Docs sets standards for feature documentation, provides clear paths for docs contribution, and coordinates documentation updates during quarterly releases.
```

Comment:
Consider adding an explicit list of the types of documentation owned by SIG docs - Documentation for the core Kubernetes APIs, for cli tools that are shipped with the Kubernetes release, etc...  e.g. SIG docs doesn't own go-doc documentation for libraries, or the sample-controller example repo.

Reply:
We should mention generation of reference documentation as well.

Reply:
+1

Reply:
I've added the specificity you recommen, @pwittrock. I'd love further review!